packages/proxy/lib/http/util/service-worker-manager.ts

```diff
@@ -120,7 +120,10 @@ export class ServiceWorkerManager {
       const activatedServiceWorker = registration.activatedServiceWorker
       const paramlessDocumentURL = browserPreRequest.documentURL.split('?')[0]
 
-      if (!activatedServiceWorker || activatedServiceWorker.initiatorURL !== paramlessDocumentURL) {
+      if (!activatedServiceWorker ||
```

Comment:
It may be beneficial to add a comment here to explain this a little bit.

Reply:
Updated: [`c4a5906` (#28748)](https://github.com/cypress-io/cypress/pull/28748/commits/c4a5906be5513a75624854f349daf80f93e850a2)